index.html

```diff
@@ -16,12 +16,12 @@
     </nav>
 </header>
 <body>
-    <h1>Hi, my name is <span class='nameColor' id='name'>Dylan</span></h1>
+    <h1 class="introTitle"><span>Hi, my name is&nbsp;</span><span class='nameColor' id='name'>Dylan</span></h1>
```

Comment:
Did you add another span tag around "Hi, my name is" so you could add the nbsp; to the existing h1. 
Also, What does the nbsp do?

Reply:
I added another span in order to do `width: 100%` easier (so that these stack on top of each other rather than side by side), I also added `&nbsp;` to replace the space after "is", since HTML will do some weirdness with spaces and try to remove it otherwise